maven-javadoc-plugin/pom.xml

```diff
@@ -193,9 +193,9 @@ under the License.
 
     <!-- misc -->
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.6</version>
```

Comment:
Since Maven 3.3+ require JDK 1.7 or above to execute is there a chance that the Java 6 requirement can be lifted any time soon? Java 6 went end of life over four years ago.